cypress/TestCases/FireboltCertification/Accessibility.feature

```diff
@@ -92,10 +92,10 @@ Feature: Accessibility
 
     @sdk @transport @Sev0
     Scenario Outline: Accessibility.voiceGuidanceSettings - Validating API and Event Responses for <Scenario>
-        Given we test the 'ACCESSIBILITY_VOICEGUIDANCE_SETTINGS' getters and setters '<Method>' to '<Value>'
+        Given the environment has been set up for 'Accessibility voiceguidance settings' tests
         And '3rd party app' registers for the 'Firebolt' event
```

Comment:
It should follow the format "Module:Method" wherever applicable or "Module Method"
        Given the environment has been set up for 'Accessibility closedCaptionsSettings' tests
        Given the environment has been set up for 'Accessibility voiceGuidanceSettings' tests

Try to follow the same format across modules - Using same name as OPEN RPC

Reply:
addressed the comment